oauth2/oauth2-core/src/main/java/org/springframework/security/oauth2/core/oidc/endpoint/OidcParameterNames.java

```diff
@@ -20,6 +20,7 @@
  * and used by the authorization endpoint and token endpoint.
  *
  * @author Joe Grandja
+ * @author Mark Heckler
```

Comment:
Please update copyright year.

Reply:
Done. Found another I'd missed & updated that too.